infrasim/package_install.py

```diff
@@ -63,9 +63,36 @@ def install_bintray_packages(repo, package):
         print "installing " + package + "..."
         run_command("dpkg -i " + file_name)
 
+def check_package(package = "Qemu", cmd = "which qemu-system-x86_64"):
+    ''' Return True if Package needs to install'''
+    install_package = False
+    # Check if package installed
+    try:
+        qemu_installed = run_command(cmd)
+    except CommandRunFailed:
+        install_package = True
+    else:
+        while True:
+            ans = raw_input(package+" already exists. Overwrite it? (Y/n)")
+            if ans.lower() not in ('yes', 'no', 'y', 'n'):
+                print("Sorry, I didn't understand that.")
```

Comment:
prompt message is not clear enough. something like "type answer with either yes, no, y or n" sounds better